<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationWidget</class>
 <widget class="QWidget" name="ApplicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>552</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layoutApps" stretch="0,1,0,0">
    <item>
     <spacer name="verticalSpacer_32">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1,0,0">
      <item>
       <spacer name="verticalSpacer_35">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::NoDragDrop</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0" rowspan="5">
           <widget class="QLabel" name="labelAppsIcon">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="pixmap">
             <pixmap resource="../zasoby.qrc">:/icons/info.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelAppsAppName">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="app name label">App name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelAppsVersion">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="version label">App version</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsFileName">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="file name label">File name</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsSize">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="size label">File Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="editAppsAppName">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="editAppsVersion">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="QLineEdit" name="editAppsFileName">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsPackageName">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="package label">Package name</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QLineEdit" name="editAppsPackageName">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="3">
           <widget class="QLineEdit" name="editAppsSize">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsLocation">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="location label">Level</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0" colspan="3">
           <widget class="QLineEdit" name="editAppsLocation">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="5">
           <widget class="QLabel" name="labelQRcode">
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_uninstall">
          <property name="text">
           <string>Uninstall</string>
          </property>
          <property name="icon">
           <iconset resource="../zasoby.qrc">
            <normaloff>:/icons/uninstall.png</normaloff>:/icons/uninstall.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_copy">
          <property name="text">
           <string>Copy To PC</string>
          </property>
          <property name="icon">
           <iconset resource="../zasoby.qrc">
            <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_install">
          <property name="text">
           <string>Install</string>
          </property>
          <property name="icon">
           <iconset resource="../zasoby.qrc">
            <normaloff>:/icons/install.png</normaloff>:/icons/install.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_refresh">
          <property name="text">
           <string>Refresh</string>
          </property>
          <property name="icon">
           <iconset resource="../zasoby.qrc">
            <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressApps">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="format">
           <string notr="true">%v/%m</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_33">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_36">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="appsLabelSelected">
      <property name="text">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_34">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
